ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -263,42 +264,61 @@
   (debug "send-update-subscription-notification-time with" data)
   (search/save-subscription-notification-time context data))
 
+(defn- filter-gran-refs-by-subscriber-id
+  "Takes a list of granule references and a subscriber id and removes any granule that the user does
+   not have read access to."
+  [context gran-refs subscriber-id]
+  (let [concept-ids (map :concept-id gran-refs)
+        permissions (json/parse-string
+                     (access-control/get-permissions context {:user_id subscriber-id
+                                                              :concept_id (map :concept-id gran-refs)}))]
+    (filter (fn [granule-reference]
+              (some #{"read"} (get permissions (:concept-id granule-reference))))
```

Comment:
I want to confirm that this check should include any granules that are accessible by registered user. Are you sure this check will include those?

Reply:
yes.

Reply:
cool. Make sure the test has a granule covers this case (i.e. accessible by registered user).